packages/wrangler/src/index.tsx

```diff
@@ -71,6 +78,19 @@ async function readConfig(configPath?: string): Promise<Config> {
   // todo: validate, add defaults
   // let's just do some basics for now
 
+  for (const binding of config.unsafe?.bindings ?? []) {
+    if ("type" in binding && typeof binding["type"] == "string") {
+      const type = binding["type"];
+      if (knownBindings.includes(type)) {
```

Comment:
I don't think we should do this check. Reason being if we ever extend these existing bindings, let's say to add a new property, they you can't test them. While I suppose it's in hypothetical land, these are unsafe after all.